internal/service/cluster_mapping.go

```diff
@@ -16,7 +16,7 @@ type ClusterMapping [][]string
 
 // IsValid check the list is in order (based on the versions), that the versions
 // can be parsed and that the remote configurations are accessible
-func (cm ClusterMapping) IsValid(remoteConfigurationPath string) bool {
+func (cm ClusterMapping) IsValid(remoteConfigurationPath string, conditionsVersion string) bool {
```

Comment:
would it make sense to set a default value of `stable`?

Reply:
I would probably still like this to be filled in explicitly, just to avoid accidental bugs (even though the function will be only called twice apart from tests, so I guess it does not matter that much).

Reply:
ok, makes sense. Thanks